Can't connect to groovy - no such file or directory

I tried installing the plugin under openSUSE 13.1 but evaluating Groovy code failed with the error:

```
find: ‘/home/gregor/.gvm/groovy/current/bin/groovy/../../embeddable’: Not a directory`
Exception in thread "main" java.lang.NoClassDefFoundError: groovy/lang/GroovyObject
    at java.lang.ClassLoader.defineClass1(Native Method)
    at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
    at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
    at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
    at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:482)
Caused by: java.lang.ClassNotFoundException: groovy.lang.GroovyObject
    at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
    at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
    ... 13 more
```

I could fix the error by editing `~/.config/LightTable/plugins/Groovy/run-server.sh` to use `dirname` to directly get the folder containing `groovy` - apparently `../` on a executable file doesn't work on my setup.

--- a/run-server.sh
+++ b/run-server.sh
@@ -6,6 +6,7 @@
 echo $3
 
 GROOVY_BIN=`which groovy`
-GROOVY_ALL_JAR=`find "$GROOVY_BIN/../../embeddable" -type f ! -name "*-indy.jar" | head -n 1`
+GROOVY_BIN_FOLDER=`dirname $GROOVY_BIN`
+GROOVY_ALL_JAR=`find "$GROOVY_BIN_FOLDER/../embeddable" -type f ! -name "*-indy.jar" | head -n 1`
 
 java -cp "$GROOVY_ALL_JAR:lib/ltgroovy.jar" lt.groovy.LTServer $1 $2 "$3"

